es-ds-docs/pages/molecules/radio-button.vue

```diff
@@ -51,51 +79,86 @@ const propTableRows = [
+            <es-radio-button-group
+                id="idFruits2"
+                label="Please choose your favorite fruit.">
+                <es-radio-button
+                    v-for="fruit in fruits"
+                    :key="fruit.key"
+                    v-model="selectedFruit"
+                    :display-name="fruit.name"
+                    group-name="stacked"
+                    :value="fruit.key" />
+            </es-radio-button-group>
         </div>
 
         <div class="my-500">
             <h2>Disabled</h2>
-            <p>Please choose your favorite fruit.</p>
-            <es-radio-button
-                v-for="fruit in fruits"
-                :key="fruit.key"
-                v-model="selectedFruit"
-                :display-name="fruit.name"
-                group-name="disabled"
-                :value="fruit.key"
-                disabled />
+            <es-radio-button-group
+                id="idFruits3"
+                label="Please choose your favorite fruit.">
+                <es-radio-button
+                    v-for="fruit in fruits"
+                    :key="fruit.key"
+                    v-model="selectedFruit"
+                    :display-name="fruit.name"
+                    group-name="disabled"
+                    :value="fruit.key"
+                    disabled />
+            </es-radio-button-group>
+        </div>
+
+        <div class="my-500">
+            <h2>Using Options Prop</h2>
+            <es-radio-button-group
+                id="test2RadioGroup"
+                v-model="test2Selected"
+                label="Radios using options"
+                :options="test2Options"
+                name="test2" />
+
+            <div>selected: {{ test2Selected }}</div>
         </div>
 
         <div class="mb-500">
             <h2>EsRadioButton props</h2>
-            <ds-prop-table :rows="propTableRows" />
+            <ds-prop-table :rows="radioButtonPropTableRows" />
+        </div>
+
+        <div class="mb-500">
+            <h2>EsRadioButtonGroup props</h2>
+            <ds-prop-table :rows="radioButtonGroupPropTableRows" />
         </div>
 
         <ds-doc-source
-            :comp-code="compCode"
-            comp-source="es-ds-components/src/lib-components/es-radio-button.vue"
+            comp-title="Radio Button Component"
+            :comp-code="radioButtonComponentCode"
+            comp-source="es-ds-components/src/lib-components/es-radio-button.vue" />
+
+        <ds-doc-source
+            comp-title="Radio Button Group Component"
+            :comp-code="radioButtonGroupComponentCode"
+            comp-source="es-ds-components/src/lib-components/es-radio-button-group.vue" />
+
+        <ds-doc-source
+            doc-title="Radio Button & Group Documentation"
             :doc-code="docCode"
-            doc-source="es-ds-docs/pages/molecules/radio-button.vue" />
+            doc-source="es-ds-docs/pages/molecules/radio-input.vue" />
```

Comment:
This should be `es-ds-docs/pages/molecules/radio-button.vue`

Reply:
😓 good catch!